integrated_channels/cornerstone/models.py

```diff
@@ -236,7 +236,9 @@ class CornerstoneLearnerDataTransmissionAudit(LearnerDataTransmissionAudit):
 
     class Meta:
         app_label = 'cornerstone'
+        # XXX non-standard
```

Comment:
is the non standard comment referring to the user and course id unique key?

Reply:
if so, is it worth adding a line about why we make sure to have one and only one pair between user and course

Reply:
I dont know why we do that, i only know that we have that constraint and its non-standard so, in theory, there is the potential we could do something that processes the abstract class and run into a constraint issue.

Reply:
it's because we pre-create the audit at the time that the student comes over to our system from cornerstone and we need a way of finding that audit at the time of grade reporting IIRC

Reply:
Altho it's not immediately obvious to me why that means we need a unique key. Maybe just to safe guard against repeated access from the student. IE if the learner comes over via external link a second time+.

Reply:
anyhoo, not necessary to add to the comment just thought it'd be good context for other people looking at the code